build.sbt

```diff
@@ -8,6 +8,8 @@ organization := "com.github.cb372"
 description := "An sbt plugin to check that your project does not directly depend on any transitive dependencies for compilation"
 licenses += ("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html"))
 
+addSbtPlugin("com.dwijnand" % "sbt-compat" % "1.2.6")
```

Comment:
Out of curiosity, why does it have to go in build.sbt?

Reply:
Because it's a dependency of the sbt plugin.  However, differently to every other sbt plugin or non-sbt plugin project, this dependency is cross-built for the different binary API of sbt 0.13 and sbt 1 (which is both sbt source API differences as well as Scala 2.10 vs 2.12 binary encoding API differences).

It's funky, but it's right.  Might want to leave some sort of comment next to it so future you (or contributors and maintainers) doesn't get confused.